tools/db_bench_tool.cc

```diff
@@ -3472,8 +3483,10 @@ void VerifyDBFromDB(std::string& truth_db_name) {
           }
         }
       }
-      if (!FLAGS_use_blob_db) {
+      if (!use_blob_db_) {
+#ifndef ROCKSDB_LITE
```

Comment:
This seems to be a bug to me. Now db_bench doesn't insert anything with LITE.

Reply:
oops, will fix.